Core/PixelEvent.swift

```diff
@@ -1024,6 +1032,15 @@ extension Pixel.Event {
         case .debugReturnUserUpdateATB: return "m_debug_return_user_update_atb"
             
         case .appRatingPromptFetchError: return "m_d_app_rating_prompt_fetch_error"
+
+        // MARK: - User behavior
+        case .userBehaviorReloadTwice: return "m_reload-twice"
```

Comment:
One other thing to check with this pixel is that if the user refreshes three times in a row, the pixel will send twice. Do you think we need some buffer to make it so that this pixel really means "user tried to reload repeatedly to fix a problem", or is it okay?